src/__tests__/HighlightText.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {render} from '@testing-library/react-native'
+import HighlightText from '../components/HighlightText/HighlightText'
+import type {ReactTestInstance} from 'react-test-renderer'
+import {StyleSheet} from 'react-native'
+
+describe('HighlightText', () => {
+  const textToHighlight = 'Lorem ipsum dolor sit amet consectetur adipiscing elit'
```

Comment:
can we use english for easy to read instead another language?

Reply:
It's just dummy text for testing